src/frontends/onnx/frontend/src/op/layer_norm.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/layer_norm.hpp"
+#include "ngraph/node.hpp"
+
+#include <algorithm>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <numeric>
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "ngraph/validation_util.hpp"
+
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/mvn.hpp"
+#include "ngraph/opsets/opset5.hpp"
+#include "ngraph/validation_util.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+//    Decompose LayerNorm(x) to MVN(x) * gamma + beta
+OutputVector layer_norm(const Node& node) {
```

Comment:
Particular operator handlers should match the ONNX operation name 100%:
```suggestion
OutputVector layer_normalization(const Node& node) {
```